product_category_company_favorite/models/product_category.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2023-Today: GRAP (<http://www.grap.coop/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    is_favorite = fields.Boolean(
+        string="Favorite",
+        company_dependent=True,
+        help="If this field is unchecked, the category will"
+        " be hidden when searching category in a drop-down list"
+        " like in the product form view."
+    )
+
+    @api.onchange("parent_id")
+    def onchange_parent_id_favorite(self):
+        if self.parent_id:
+            self.is_favorite = self.parent_id.is_favorite
```

Comment:
Less important than the other for me, but as seen by codecov, a small test is missing here.

Reply:
Done in ``test_40_test_onchange``